CRM/Contribute/Form/ContributionPage/Amount.php

```diff
@@ -518,7 +518,7 @@ public function postProcess() {
         'calendar_month' => 'pledge_calendar_month',
       );
       if ($params['pledge_default_toggle'] == 'contribution_date') {
-        $fieldValue = json_encode(array('contribution_date' => date('Ymd')));
+        $fieldValue = json_encode(array('contribution_date' => date('m/d/Y')));
```

Comment:
This smells a bit to me since there are settings for how dates are formatted. Just because the default format is American m/d/Y doesn't mean this will work in Canada for Ymd or d/m/Y formats. I'm going to see about creating an instance to check this.